translations/messages.fr.yaml

```diff
@@ -1 +1,31 @@
-Submit: Enregistrer
+action:
+  login: Connexion
+  loginSubmit: Je me connecte
+  logout: Déconnexion
+  rememberMe: Se souvenir de moi
+  submit: Enregistrer
+calendar:
+  week:
+    current: Semaine actuelle
+    next: Semaine prochaine
+  from: du
```

Comment:
the only thing we could improve here could be using a single `Du %from% au %to%` translation instead of `t1 +from +t2+to`